docs/tutorials/wiki/installation.rst

```diff
@@ -317,7 +317,7 @@ On Unix
 
 .. code-block:: bash
 
-    $VENV/bin/pytest --cov=tutorial tests.py -q
+    $VENV/bin/pytest --cov=tutorial tests -q
```

Comment:
your invocations of pytest are inconsistent across chapters. The folder does not need to be specified and notice above on line 264/272 you used `--cov` not `--cov=tutorial`.

Reply:
Check the context for this invocation:

> Without these defaults, we would need to specify the path to the module on which we want to run tests and coverage.

To verify, I moved both the default `pytest.ini` and `.coveragerc` out of the project directory.

```
$VENV/bin/pytest --cov=tutorial -q
11 errors in 5.41s

$VENV/bin/pytest --cov=tutorial tests -q
# succeeds
```

Invocations of pytest in `wiki/installation.rst` were copied from `wiki2/installation.rst` verbatim.

I did find a couple of other unrelated things that needed to be updated, though, and pushed those commits.

Reply:
we have the testpaths set in pytest.ini, you do not need to specify the tests folder.

Reply:
Then we need to remove the context as well from both tutorials because it would repeat an earlier step.

https://github.com/Pylons/pyramid/pull/3557/files/56fd7cbeaf107405cca2399d2b0bba60c8e60010..04cef9070f8554a9a025ae968368d03b4ed939c2#diff-11018ed84544958d5baea4415d2639edR307-R320

I think it is useful to retain this step because it demonstrates the verbosity of commands when you do not have the default configuration files. Your call.

Reply:
well the cookiecutter output just says to run `pytest` and the point of that config was to keep the calling convention simple so I'd rather it minimize the number of cli args being passed

Reply:
Yes, I understand the purpose of adding the config updates.

I am asking whether you want to remove these two sections, which explain what one needs to do without those two config files (and implies why config files are a good thing™), as well as links to more pytest options:

https://github.com/Pylons/pyramid/pull/3557/files/56fd7cbeaf107405cca2399d2b0bba60c8e60010..04cef9070f8554a9a025ae968368d03b4ed939c2#diff-11018ed84544958d5baea4415d2639edR305-R335

https://github.com/Pylons/pyramid/pull/3557/files#diff-e0805d6d677c4e8151d97c2a617fe1dcR384-R416

Reply:
Yes I think I'd prefer to remove the section saying "if things weren't this way then you'd have to do this" because they can just appreciate our config and read the pytest docs themselves. I'd keep the part mentioning that the config files exist.

Reply:
OK, please check out commit 3152aa5.